HopsanGUI/LibraryHandler.h

```diff
@@ -84,6 +113,46 @@ class ComponentLibraryEntry
     SharedComponentLibraryPtrT pLibrary;        //!< What library the object belongs to
     HiddenVisibleEnumT visibility=Hidden;       //!< Visible or hidden in library widget
     QStringList displayPath;                    //!< The display path in the library widget
+    EnabledDisabledEnumT disabled=Enabled;      //!< Is component enabled or disabled?
```

Comment:
This variable name is confusing

Reply:
It did make sense when it was a Boolean, but I changed to enum to make it less confusing :man_facepalming: